### PR TITLE
doc: how to use and hack Cozy UI components 📚

### PR DESCRIPTION
**Pull request by gregorylegarec**
As I was diving into Cozy-UI for the first time since a long time ago, I was looking for a documentation about how to develop components, and did not find anything. So I used the only way I know and this PR document it, if any new developer or contributor is interested.

### CODE REVIEW

File: react/HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# How to integrate Cozy UI to your application
+## Add Cozy-UI as a dependency
```
Comment (ptbrowne) [inline react/HOWTO.md]: I would merge those 2 titles

File: react/HOWTO.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# How to integrate Cozy UI to your application
```
Comment (ptbrowne) [inline react/HOWTO.md]: Or just How to

File: react/HOWTO.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Use [`yarn`](https://yarnpkg.com/) to add Cozy UI to a dependency to your project.
+
+```bash
+$ yarn add cozy-ui
+```
+
+You're now ready to use [Cozy UI's (p)React components](https://cozy.github.io/cozy-ui/react/)
+
+## Hack Cozy UI
```
Comment (ptbrowne) [inline react/HOWTO.md]: Hacking Cozy UI is not part of "How to integrate Cozy UI to your application" so I would change the titles to reflect this.

File: react/HOWTO.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# How to integrate Cozy UI to your application
+## Add Cozy-UI as a dependency
+
+Use [`yarn`](https://yarnpkg.com/) to add Cozy UI to a dependency to your project.
+
+```bash
+$ yarn add cozy-ui
```
Comment (ptbrowne) [inline react/HOWTO.md]: I think this step should be in the README.md and should replace the stylus plugin section that no one uses.

File: react/HOWTO.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+Then in your application folder, you can link to your local Cozy UI
+
+```bash
+$ yarn link cozy-ui
+$ yarn watch
+```
+
+All your modification in your local Cozy UI will now be visible in your application!
```
Comment (ptbrowne) [inline react/HOWTO.md]: I think this could go in the main README.md also.

**Comment by gregorylegarec** [comment]
Thanks for your feedback @ptbrowne, actually I was asking myself if this doc deserved to be in the main README. I am glad you're in favor of it 👍

**Comment by gregorylegarec** [comment]
@ptbrowne I've merged all the modifications in the README.md, I choose to keep the stylus part, I think it could still be useful. If you're ok with the README structure, feel free to merge.

**Comment by ptbrowne** [comment]
> I choose to keep the stylus part, I think it could still be useful

To whom ? I personnally do not want to advertise for a 3rd way of integrating Cozy UI into a project. It is confusing. To help people understand, we need to guide them and have to make decisions on what we support and what we don't. I personnally have never seen a Cozy project that uses only the Stylus plugin so I cannot give support on this.

**Comment by gregorylegarec** [comment]
Honestly I don't know, but the process described here is stil used in [Collect](https://github.com/cozy/cozy-collect/blob/master/src/ducks/connections/components/queue/styles.styl#L1), so I feel that this part is still relevant. As this is not really the point of this PR, what about dealing with the stylus part in another one ?

**Comment by ptbrowne** [comment]
OK, let's do this in another PR.

**Pull request merged**